src/hooks/useSubmitTxn.ts

```diff
@@ -122,6 +271,29 @@ export default () => {
         throw new Error(t('errors.account'))
       }
 
+      if (!walletSignBottomSheetRef) {
+        throw new Error('No wallet sign bottom sheet ref')
+      }
+
+      const serializedTx = txns[0].serialize({
+        requireAllSignatures: false,
+      })
+      setSerializedTx(serializedTx)
```

Comment:
It's always only setting one tx throughout this file. What if there are multiple?